python/QC3.py

```diff
@@ -0,0 +1,31 @@
+from subprocess import CalledProcessError
+from wrappers import runCommand
+from options import parser
+
+args = parser.parse_args()
+cmd = ["python","$GEM_BASE/python/Produce_Config_File.py"]
+
+for f in args.file:
+	cmd.append(f)
+	pass
+
+cmd.append("--CanvTitleX=Time #left(s#right)")
+cmd.append("--CanvTitleY=Pressure #left(mbar#right)")
+cmd.append("--SelectColumnX= 1")
+cmd.append("--SelectColumnY= 2")
+cmd.append("--SelectRowStart= 1")
+cmd.append("--SelectRowEnd= 61")
+cmd.append("--CanvRangeX= 0,3600")
+cmd.append("--CanvRangeY= 0,35")
+
+
+if len(args.file)==1:
+	filetype=f[int(f.index('.')):]
+	filename = f
+	filename= filename.replace(filetype,"")
+	cmd.append("--OutputName="+filename)
```

Comment:
Yes I am sure it doesn't over-write the input file name (tested)..It's just finds the type of the file (.xlsm for example) and replaces it with an empty string.

I have changed the names in the output files as requested.